pymeasure/instruments/keithley/keithley2000.py

```diff
@@ -430,11 +430,6 @@ class Keithley2000(Instrument, KeithleyBuffer):
         values=[0, 999999.999]
     )
 
-    def __init__(self, adapter, **kwargs):
-        super(Keithley2000, self).__init__(
-            adapter, "Keithley 2000 Multimeter", **kwargs
```

Comment:
Isn't the init still necessary to set the instrument name?

Reply:
Thank you for pointing this out, I was a bit too quick there